Classes/Util.php

```diff
@@ -114,7 +114,7 @@ public static function isDraftRecord(string $table, int $uid): bool
         $isWorkspaceRecord = false;
 
         if ((ExtensionManagementUtility::isLoaded('workspaces')) && (BackendUtility::isTableWorkspaceEnabled($table))) {
-            $record = BackendUtility::getRecord($table, $uid, 'pid, t3ver_state');
+            $record = BackendUtility::getRecord($table, $uid, 'pid, t3ver_state', '', false);
 
             if ($record['pid'] == '-1' || $record['t3ver_state'] > 0) {
```

Comment:
Officially getRecord() can return either array or null. This patch works, but I would prefer to check for array first, before accessing the array keys.